docs/install.rst

```diff
@@ -40,11 +40,17 @@ supports
 <https://github.com/certbot/certbot/blob/master/certbot-apache/certbot_apache/constants.py>`_
 modern OSes based on Debian, Fedora, SUSE, Gentoo and Darwin.
 
+
+To double check the integrity of the ``certbot-auto`` script before running it using gpg2
+requires the package gnupg or gnupg2 installed
```

Comment:
How would you feel about changing the wording to something like:

Additional integrity verification of certbot-auto script can be done by verifying its digital signature. 
This requires a local installation of gpg2, which comes packaged in many Linux distributions under name gnupg or gnupg2.

Reply:
Ya 
That looks more clear